permits/templates/permits/permit_request_new.html

```diff
@@ -1,8 +1,12 @@
 {% extends "base_generic.html" %}
 {% load bootstrap4 %}
 {% load i18n %}
+{% block navbar %}
+{% load permit_progressbar %}
```

Comment:
Move the `{% load permit_progressbar %}` out of the navbar block (put it with the other `load` tags).